phone_verify/backends/kavenegar.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+# Third Party Stuff
+from kavenegar import KavenegarAPI, APIException, HTTPException
+
+# Local
+from .base import BaseBackend
+
+
+class KavenegarBackend(BaseBackend):
+    def __init__(self, **options):
+        super(KavenegarBackend, self).__init__(**options)
+        # Lower case it just to be sure
+        options = {key.lower(): value for key, value in options.items()}
+        self.api_key = options.get("secret", None)
+        self.sender = options.get("from", None)
+
+        self.client = KavenegarAPI(self.api_key)
+        self.exception_class = APIException, HTTPException
+
+    def send_sms(self, number, message):
+        params = {'receptor': number, 'template': '', 'token': message, 'type': 'sms'}
+        self.client.sms_send(params)
+
+    def send_bulk_sms(self, numbers, message):
+        params = {'sender': self.sender, 'receptor': numbers, 'message': message, }
```

Comment:
The API here states that sender should be a list of strings and you're providing a single sender here 🤔 

https://github.com/kavenegar/kavenegar-python

Reply:
Hi @SepehrHasanabadi 

Can you please have a look at this when you get the time?

Reply:
Ping @SepehrHasanabadi Can you please have a look at this conversation? Have you tested these changes?

Reply:
Also, @SepehrHasanabadi, have a look at this comment as well.